### PR TITLE
Initial creation of cdi-3.0

### PR DESCRIPTION
**Pull request by tevans78**
This is the initial creation of the cdi-3.0 feature and bundles. Enabled the feature to start ... but isn't really working yet.

### CODE REVIEW

**Comment by tevans78** [comment]
#build

**Comment by tevans78** [comment]
#build

**Comment by andrea-pichler** [comment]
No message review requested from L2

File: dev/com.ibm.websphere.appserver.features/visibility/public/cdi-3.0/resources/l10n/com.ibm.websphere.appserver.cdi-3.0.properties [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=The Contexts and Dependency Injection 3.0 specification makes it easier to integrate \
+ Jakarta EE components of different types. It provides a common mechanism to inject component \
```
Comment (jantley-ibm) [inline dev/com.ibm.websphere.appserver.features/visibility/public/cdi-3.0/resources/l10n/com.ibm.websphere.appserver.cdi-3.0.properties:18]: change
component \
to
components \

File: dev/com.ibm.websphere.appserver.features/visibility/public/cdi-3.0/resources/l10n/com.ibm.websphere.appserver.cdi-3.0.properties [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+ such as EJBs or Managed Beans into other components such as JSPs or other EJBs.
```
Comment (jantley-ibm) [inline dev/com.ibm.websphere.appserver.features/visibility/public/cdi-3.0/resources/l10n/com.ibm.websphere.appserver.cdi-3.0.properties:19]: change
Managed Beans
to
managed beans

**Comment by tevans78** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/cdi-3.0/resources/l10n/com.ibm.websphere.appserver.cdi-3.0.properties (hunk 1 above)
Comment (tevans78) [reply dev/com.ibm.websphere.appserver.features/visibility/public/cdi-3.0/resources/l10n/com.ibm.websphere.appserver.cdi-3.0.properties]: done

File: dev/com.ibm.websphere.appserver.features/visibility/public/cdi-3.0/resources/l10n/com.ibm.websphere.appserver.cdi-3.0.properties (hunk 2 above)
Comment (tevans78) [reply dev/com.ibm.websphere.appserver.features/visibility/public/cdi-3.0/resources/l10n/com.ibm.websphere.appserver.cdi-3.0.properties]: done

**Comment by tevans78** [comment]
#build
#run-libby-bot

**Comment by tevans78** [comment]
#build
#run-libby-bot

**Pull request closed without merging**